xml/System.Text.Json/JsonReaderOptions.xml

```diff
@@ -84,10 +95,21 @@ By default, the reader throws a <xref:System.Text.Json.JsonReaderException> if i
         <ReturnType>System.Int32</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets or sets the maximum depth allowed when reading JSON, with the default (i.e. 0) indicating a max depth of 64. Reading past this depth will throw a <exception cref="T:System.Text.Json.JsonException" />.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets or sets the maximum depth allowed when reading JSON, with the default (i.e. 0) indicating a maximum depth of 64. </summary>
```

Comment:
I learned recently that we shouldn't use i.e., from @mairaw https://github.com/dotnet/corefx/pull/39092#discussion_r304071760

Reply:
Change to `that is,`, like what @safern did here: https://github.com/dotnet/corefx/pull/39092/commits/f20cfb8e1ab78790426fda2d7efbee763d3398c1